gax/test/unit/regapic.ts

```diff
@@ -569,4 +583,86 @@ describe('REGAPIC', () => {
       }, /* catch: */ done);
     });
   });
+
```

Comment:
should we also add a test that confirms the response was minified? it seems like we're testing data integrity and that we're sending the prettyPrint request, but maybe we could also see if the actual size of the response is smaller. We have a similar test [here](https://github.com/googleapis/gax-nodejs/blob/47417e4d97a43b4cbb2c4ac16e574201b65edc43/tools/test/minify.ts) that could be useful!

Reply:
I agree that this test would be useful, but I think it requires a system test that hits a real service endpoint. It's not clear to me how to implement that.